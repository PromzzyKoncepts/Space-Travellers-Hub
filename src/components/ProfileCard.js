/* eslint-disable react/prop-types */
import React from 'react';

function ProfileCard({ name }) {
  return (
    <div className="profileCard">
      {
        name
      }
    </div>
  );
}

export default ProfileCard;
