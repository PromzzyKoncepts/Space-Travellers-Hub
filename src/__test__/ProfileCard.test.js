import React from "react";
import renderer from "react-test-renderer";
import { Provider } from "react-redux";
import store from "../Redux/ConfigureStore";
import ProfileCard from "../components/ProfileCard";

it("renders Component correctly", () => {
  const tree = renderer.create(
    <Provider store={store}>
      <ProfileCard />
    </Provider>
  );
  expect(tree).toMatchSnapshot();
});
