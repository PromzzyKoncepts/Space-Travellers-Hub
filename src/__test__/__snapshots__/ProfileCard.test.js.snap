// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Component correctly 1`] = `
<div
  className="profileCard"
/>
`;
